### PR TITLE
Update service.rs

### PR DESCRIPTION
**Pull request by pjeziorowski**

### CODE REVIEW

**Review by erebe — APPROVED** [review]

File: src/cloud_provider/service.rs [hunk 1]
```diff
@@ -333,8 +333,8 @@ where
         service.engine_error(
             EngineErrorCause::User(
                 "Your application has failed to start. \
-                Ensure you can run it without issues with `qovery run` and check its logs from the web interface or the CLI with `qovery log`",
-            ),
+                Ensure you can run it without issues with `qovery run` and check its logs from the web interface or the CLI with `qovery log`.\
+                HINT: this issue often occurs due to ports misconfiguration. Make sure you exposed the correct port (using EXPOSE statement in Dockerfile or via Qovery configuration). ",
```
Comment (evoxmusic) [inline src/cloud_provider/service.rs:337]: nice one @pjeziorowski 👍🏽

**Review by evoxmusic — APPROVED** [review]

File: src/cloud_provider/service.rs (hunk 1 above)
Comment (deimosfr) [reply src/cloud_provider/service.rs:337]: This message is already the HINT message. Please don't provide a HINT into a HINT, I propose you to rephrase it to make it more precise and concise.
Also, it can't compile actually

**Comment by deimosfr** [comment]
I've updated it: https://github.com/Qovery/engine/commit/0bd53c3adc6bacf248c273aa256c2864e5e75003
